src/skins/window.cc

```diff
@@ -93,6 +93,7 @@ Window::Window (int id, int * x, int * y, int w, int h, bool shaded) :
     h *= config.scale;
 
     GtkWidget * window = gtk_window_new (GTK_WINDOW_TOPLEVEL);
+    gtk_window_set_role ((GtkWindow *) window, "skin");
```

Comment:
The role should be set by the subclasses (`mainwindow` or `playlist` or `equalizer`) rather than by the base class.

Reply:
This still needs to be fixed.